### DISCUSSION TITLE
2022/07/27/Mysql%E4%B8%BB%E4%BB%8E%E5%90%8C%E6%AD%A5%E7%AB%9F%E7%84%B6%E8%BF%99%E4%B9%88%E5%A4%9A%E7%BB%86%E8%8A%82/

### DISCUSSION THREAD

**Discussion by giscus** [skitii/skitii#8] [General]
# 2022/07/27/Mysql%E4%B8%BB%E4%BB%8E%E5%90%8C%E6%AD%A5%E7%AB%9F%E7%84%B6%E8%BF%99%E4%B9%88%E5%A4%9A%E7%BB%86%E8%8A%82/

踩踩Mysql主从同步的坑

https://skitii.vercel.app/2022/07/27/Mysql%E4%B8%BB%E4%BB%8E%E5%90%8C%E6%AD%A5%E7%AB%9F%E7%84%B6%E8%BF%99%E4%B9%88%E5%A4%9A%E7%BB%86%E8%8A%82/

**Comment by AbnerHuang2** [comment]
细节

**Comment by AbnerHuang2** [comment]
为什么这个评论这么加载这么慢呀，哭了

**Comment by Tony10086** [comment]
111

**Reply by AbnerHuang2** [reply]
你是哪个sb

**Comment by Tony10086** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    
        
            
        
    
    我是黄后盛
    
      
    
        
                    
            
                
                    
                            
                                
                            
                            
                                1611797218
                            
                    
                        
                            
                                    ***@***.***
                            
                        
                
            
        
        
    
 
    

  
    ---- Replied Message ----
  
  
    
      
        
         From 
        
        
            ***@***.***>
            
        
      
      
        
         Date 
        
        
        07/28/2022 10:45
        
      
      
        
         To 
        
        
         
          
            ***@***.***>
            
          
        
      
      
        
         Cc 
        
        
          
            ***@***.***>
            ,
          
          
            ***@***.***>
            
          
        
      
      
        
         Subject 
        
        
              Re: [skitii/skitii] 2022/07/27/Mysql%E4%B8%BB%E4%BB%8E%E5%90%8C%E6%AD%A5%E7%AB%9F%E7%84%B6%E8%BF%99%E4%B9%88%E5%A4%9A%E7%BB%86%E8%8A%82/ (Discussion #8)
        
      
    
  
  

你是哪个sb



—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>
